lib/fennec/stun.ex

```diff
@@ -3,18 +3,29 @@ defmodule Fennec.STUN do
   # Processing of STUN messages
 
   alias Fennec.TURN
+  alias Jerboa.Params
   alias Fennec.Evaluator
   alias Fennec.UDP
+  alias Fennec.Auth
 
   @spec process_message(binary, Fennec.client_info, UDP.server_opts, TURN.t) ::
     {:ok, {binary, %TURN{}}} | {:ok, :void}
   def process_message(data, client, server, turn_state) do
-    with {:ok, params} <- Jerboa.Format.decode(data),
+    with secret        <- Auth.get_secret(),
```

Comment:
You can replace `<-` with `=` here. It saves the compiler on generating one extra `case` expression